README.md

```diff
@@ -39,7 +39,7 @@ If you have any questions, feel free to create a [discussion](https://github.com
 > [!NOTE]
 > **The tool is in an experimental stage, which is subject to frequent changes, and has limited documentation.**
 > We are working on improving that.
-> In the meantime, users can explore how to use the framework via a [Tutorial](https://mqt.readthedocs.io/projects/qudits/tutorial), showcasing its main functionality.
+> In the meantime, users can explore how to use the framework via a [Tutorial](https://mqt--13.org.readthedocs.build/projects/qudits/en/13/tutorial.html), showcasing its main functionality.
```

Comment:
That it is not the correct link to use. This just points to a temporary build for the PR. You want to be using: 
```suggestion
> In the meantime, users can explore how to use the framework via a [Tutorial](https://mqt.readthedocs.io/projects/qudits/en/latest/tutorial.html), showcasing its main functionality.
```